cli/command/container/logs.go

```diff
@@ -78,6 +63,11 @@ func runLogs(dockerCli *command.DockerCli, opts *logsOptions) error {
 	}
 	defer responseBody.Close()
 
+	c, err := dockerCli.Client().ContainerInspect(ctx, opts.container)
+	if err != nil {
+		return err
+	}
```

Comment:
I thought it would make sense to move this later in the function, but this changes the output a bit, because the error is now returned by the daemon (line 62 above);

```
Error response from daemon: No such container: testlogsnocontainer
```

instead of

```
Error: No such container: testlogsnocontainer
```

I can change back if we don't think this should be changed